nuxeo-compound-documents-web/ftest/features/step_definitions/browse.js

```diff
@@ -0,0 +1,11 @@
+import { Then } from '@cucumber/cucumber';
+
+Then('I can see the {string} {word} tree node is selected', function (title, tab) {
+  this.ui.drawer._section(tab).waitForVisible();
+  driver.waitUntil(() =>
+    this.ui.drawer
+      ._section(tab)
+      .elements('.content span[class*="compound"].selected a')
```

Comment:
In fact, we shouldn't probably have the `tab` parameter here, because I'm not sure how reusable this step would be.
Furthermore, perhaps we don't even need to test that the tree node is selected, as we already do it on the unit tests, wdyt? Either way, we can still keep the assert here, but simplify the step, wdyt?

Reply:
I propose to change the step to:
```
Then('I can see the {string} browser tree node is selected', function(title)`
```
Which makes more sense to our use cases, wdyt?

Reply:
This only works for the compound document tree, right? If so, we should probably include that in the description as well. Or we could remove the `span[class*="compound"]` and make it mire generic, for example.

Reply:
I think we could remove the `span[class*="compound"]` and make it more generic.